### PR TITLE
Adds Github section to style guide

### PR DESCRIPTION
**Pull request by iteles**
Adds sections on:
- [x] Writing issues
- [x] Writing commit messages
- [x] Writing pull requests
- [x] Naming repos

Fixes https://github.com/dwyl/style-guide/issues/2

### CODE REVIEW

**Comment by iteles** [comment]
@nelsonic Please take a look and make any amendments :blush: Thanks!

File: README.md [hunk 1]
```diff
@@ -59,6 +66,61 @@ else {
...
+
+<img width="723" alt="paired-commit-message" src="https://cloud.githubusercontent.com/assets/4185328/9212496/3adc19c2-4081-11e5-956c-8655c1f37946.png">
++ Did you know you can [use emojis in your commit messages](https://github.com/dwyl/start-here/issues/49)? It's totally a thing.
+<img width="695" alt="emojis-in-commit-messages" src="https://cloud.githubusercontent.com/assets/4185328/9212573/14112b74-4082-11e5-822e-bc66250c5712.png">
+
+
+
+###Pull Requests
+Good pull requests (PR) reduce the back and forth required between the person submitting the PR and the assigned reviewer, saving everyone time.
++ Before submitting a PR, **open a new issue** so you can _confirm the need_ for the functionality you are intending to add (like in [this example](https://github.com/dshaw/env/issues/6))
```
Comment (nelsonic) [inline README.md]: I prefer this PR: https://github.com/indexzero/ps-tree/pull/12

Comment (iteles) [inline README.md]: Changed to the correct issue :+1:

**Comment by nelsonic** [comment]
Thanks @iteles 
Great start! :+1:

**Pull request merged**